src/main/resources/static_config/static_action_groups.yml

```diff
@@ -233,8 +233,10 @@ manage_point_in_time:
   static: true
   allowed_actions:
     - "indices:data/read/point_in_time/create"
-    - "cluster:admin/point_in_time/delete"
-    - "cluster:admin/point_in_time/read*"
+    - "indices:data/read/point_in_time/delete"
+    - "indices:data/read/point_in_time/readall"
+    - "indices:data/read/search"
+    - "cluster:admin/point_in_time/read_from_nodes"
```

Comment:
Sorry, for my understanding what is `read_from_nodes`?

Reply:
This is the parent action that basically reads all active Point in time searches from nodes. 

This the action for NodesGetAllPitAction.

We can rename it as well, if the name is confusing.